src/e2e/app.ts

```diff
@@ -107,6 +107,17 @@ export class App {
       await dialog.accept();
     });
 
+    const logsPath =
+      "reports/screenshots/" + expect.getState().currentTestName + ".txt";
+    page.on("console", (msg) => {
+      fs.appendFile(logsPath, msg.text(), (err) => {
```

Comment:
If you look at the published files they seem to have all the text on one line.
Perhaps we need to add a newline character here?

Reply:
[`109f052`](https://github.com/microbit-foundation/python-editor-next/pull/474/commits/109f052b09bee1277305477bdc62a428562bf793)

Reply:
Change looks good. Will just review the output then merge.